minard/channeldb.py

```diff
@@ -3,6 +3,7 @@
 from .views import app
 import psycopg2
 import psycopg2.extensions
+import detector_state
```

Comment:
Should be `from . import detector_state`

Reply:
I'm not sure the best way to do this since channeldb imports detector state the "from" syntax didn't seem to work, at least at the top of the page. I added the import to the relevant function.

Reply:
Ah this is a circular import problem since detector state imports from channeldb and channeldb imports from detector state.